tools/variant_simulator/simulate_variation

```diff
@@ -34,30 +34,33 @@ use Bio::EnsEMBL::Registry;
 use Getopt::Long qw(GetOptions);
 
 my ($help, $chromOnly, $gene, $registryFile,
-$exonOnly, $intronOnly, $codingOnly,$refseq_genes
+$exonOnly, $intronOnly, $codingOnly,$onlyMane,$refseq_genes,$spliceai
 );
 my ($species, $assembly, $edges, $outFile) = ("homo_sapiens", "grch38", 0, "simulated.vcf");
 
 usage() if (!scalar(@ARGV));
 
 GetOptions( "species=s" => \$species,
     "chrom=s" => \$chromOnly,
-    "gene=s" => \$gene,
-    "edge=i" => \$edges,
-    "output|o=s" => \$outFile,
-    "exonsOnly" => \$exonOnly,
-    "codingOnly" => \$codingOnly,
+    "gene=s"  => \$gene,
+    "edge=i"  => \$edges,
+    "output|o=s"  => \$outFile,
+    "exonsOnly"   => \$exonOnly,
+    "codingOnly"  => \$codingOnly,
+    "onlyMane"    => \$onlyMane,
     "intronsOnly" => \$intronOnly,
-    "assembly=s" => \$assembly,
-    "refseq"     => \$refseq_genes,
-    "registry=s" => \$registryFile,
+    "assembly=s"  => \$assembly,
+    "refseq"      => \$refseq_genes,
+    "spliceai"    => \$spliceai,
+    "registry=s"  => \$registryFile,
     "help!" => \$help,
 ) or usage();
 
 usage() if ($help);
 
 die "Illegal arguments -exonsOnly -intronsOnly can't be used together !" if $exonOnly && $intronOnly;
 die "Illegal arguments -intronsOnly -codingsOnly can't be used together !" if $codingOnly && $intronOnly;
+die "Illegal arguments -spliceai -intronsOnly -codingsOnly -exonsOnly can't be used together !" if $spliceai && $intronOnly && $exonOnly && $codingOnly;
```

Comment:
This will only work if all four options are used at same time, which already happen at previous die. All those four options should be triggered together, or any pairwise from those four should not be triggered?

Reply:
I fixed the condition. -spliceai can be used with the other ones, only mane can't.

Reply:
Thanks! Solved.